File: src/endToEndTest/java/com/github/miyashy/karatesample/controller/RunAllTest.java
package com.github.miyashy.karatesample.controller;

import com.intuit.karate.Results;
import com.intuit.karate.Runner;
import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Tag;
import org.junit.jupiter.api.Test;

/**
 * 全件実行用のRunner
 */
public class RunAllTest {

    @Tag("all")
    @Test
    void runAllTest() {
        Results results = Runner.path("classpass:").tags("~@ignore").parallel(1);
        Assertions.assertEquals(0, results.getFailCount());
    }
}

File: src/test/kotlin/com/github/miyashy/karatesample/controller/RunAllTest.kt
package com.github.miyashy.karatesample.controller

import com.intuit.karate.Runner
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Tag
import org.junit.jupiter.api.Test


/**
 * 全件実行用のRunner
 */
class RunAllTest {

    @Tag("all")
    @Test
    fun runAllTest() {
        val results = Runner.path("classpath:").tags("~@ignore").parallel(2)
        Assertions.assertEquals(0, results.failCount)
    }
}